frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ManageServingRuntimeModal.tsx

```diff
@@ -64,16 +66,17 @@ const ManageServingRuntimeModal: React.FC<ManageServingRuntimeModalProps> = ({
 
   const tokenErrors = createData.tokens.filter((token) => token.error !== '').length > 0;
 
+  const resourceIsSet = (fieldName, min = 1) => {
```

Comment:
This is going to conflict with my PR in #937 -- it got a significant overhaul to handle units.